pkgs/development/libraries/libvmi/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, xen, which, fetchFromGitHub, yacc, bison, flex, glib, libtool, autoconf, automake, pkgconfig, libvirt }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "libvirt-${version}";
+  version = "20170527-${stdenv.lib.strings.substring 0 7 rev}";
```

Comment:
we prefer to have a numeric value at the end, to make it sortable for `nix-env -u`. See also https://nixos.org/nixpkgs/manual/#sec-package-naming